### ISSUE TITLE
Files with “VOSTFR” in them have the wrong language detected 

### ISSUE THREAD

**Issue by CerealMs** [Sonarr/Sonarr#5751]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Some releases with “VOSTFR” in the name are detected as English releases :

![IMG_8435](https://github.com/Sonarr/Sonarr/assets/81803035/c960bced-2e4d-46ae-a36c-ce5992e25ab3)

### Expected Behavior

The releases that have “VOSTFR” in the name should be detected at French 

### Environment

```markdown
- OS: Unraid 6.12.0
- Sonarr: 3.0.10.1567
- Docker Install: yes
- Using Reverse Proxy: no
```

### What branch are you running?

Main

### Trace Logs?

[sonarr.txt](https://github.com/Sonarr/Sonarr/files/11796144/sonarr.txt)

**Comment by markus101** [comment]
Duplicate of https://github.com/Sonarr/Sonarr/issues/4763

**Comment by CerealMs** [comment]
“VOSTFR Does not mean French Audio so there is no rational reason to assume it is French language, hence will not happen
VOSTFR stands for Version Originale Sous-Titrée en FRançais, meaning that it is the original version of a film with subtitles in French superimposed onto the screen”

it is a big issue when the “original voice” is in Japanese and the only subtitles are in French because of that parsing…

**Comment by CerealMs** [comment]
> Duplicate of #4763

it is a big issue when the “original voice” is in Japanese and the only subtitles are in French because of that parsing…

**Comment by markus101** [comment]
That behaviour has already changed in v4 (in beta), instead of assuming English when the language isn't included it uses the original language of the series.

VOSTFR would be Japanese in that case, not English.
